.github/workflows/pre_release.yaml

```diff
@@ -0,0 +1,105 @@
+name: Create a pre-release
```

Comment:
This flow should really have a concurrency group and either
- let only the latest commit run
- wait for existing runs before proceeding

Reply:
Yeah, excellent point. I've seen many different takes on this in our projects. Do you know which is the favored approach?

Reply:
I think it makes sense to finish the publishing for canary builds that already started. We do cancel the tests workflow, and I'd say that's fine, but if we already started with the canary build, we should finish that. So I would go with the second option personally.

Reply:
Sure, makes sense. Do we already do that in some of our workflows?

Reply:
Probably not, I honestly don't know how this can be done, I only ever dealt with canceling, not the other way around. @vladfrangu do you have something specific on your mind outside of removing the cancel action (which I would rather not)?

Reply:
I'll get that to you in a few!

Reply:
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#concurrency

Just set cancel-in-progress to false and pick a group (probs unique)

Reply:
Nice, sounds like we could also replace the `styfle/cancel-workflow-action` action we use everywhere with this native solution then?

Reply:
Its likely, yes! But we'd need to be careful, for some flows we should still use the concurrency with cancel-in-progress set to true (like the testing on platform ones)

Reply:
yeah, lets leave that for later, we have better things to do than breaking things that work :D